release-notes/opensearch.release-notes-3.0.0-alpha1.md

```diff
@@ -0,0 +1,134 @@
+## 2025-02-27 Version 3.0.0-alpha1 Release Notes
+
+## [3.0.0-alpha1]
```

Comment:
This should probably not use the alpha1 qualifier. These will ultimately be the release notes for the official 3.0 release.

Reply:
Removed the qualifier